Makefile

```diff
@@ -548,11 +548,6 @@ docker-isolated:
 	docker start tools_awx_1
```

Comment:
you didn't get an error that the create command was deprecated? This command didn't work for me when I tried it.

but the rest of these changes look good.

Reply:
I'm using the latest `docker-ce` on Fedora and this seems to work for me:

```
docker --version
Docker version 18.09.0, build 4d60db4
```

Fresh `make docker-isolated` for me:

<img width="376" alt="image" src="https://user-images.githubusercontent.com/214912/49126061-49131580-f290-11e8-8e74-1ea2a0af09d9.png">